040-refactor-client-examples.md

```diff
@@ -0,0 +1,70 @@
+# Refactor KafkaConfig files in Strimzi Client Examples
+
+This proposal is about simplifying the `Kafka*Config` files of the Java modules of the [Strimzi Client Examples repo](https://github.com/strimzi/client-examples).
```

Comment:
Is `Kafka__<example>__Config` clearer than `Kafka*Config`?
underscores italicizing the <example> replaceable bit

Reply:
I would like to keep it as `Kafka*Config` but if you feel strongly I will change it!